gtsam/sfm/BinaryMeasurement.h

```diff
@@ -45,11 +45,43 @@ template <class T> class BinaryMeasurement : public Factor {
   T measured_;                  ///< The measurement
   SharedNoiseModel noiseModel_; ///< Noise model
 
-public:
+ public:
   BinaryMeasurement(Key key1, Key key2, const T &measured,
-                    const SharedNoiseModel &model = nullptr)
-      : Factor(std::vector<Key>({key1, key2})), measured_(measured),
-        noiseModel_(model) {}
+                    const SharedNoiseModel &model = nullptr,
+                    bool useHuber = false)
```

Comment:
I'm a bit puzzled about this flag. Why not just give BinaryMeasurement a robust error model as the input argument. I think the code should be re-factored to do just that, construct the noise model out of this class.